modules/compiler/vecz/test/lit/llvm/ScalableVectors/define_subgroup_scans.ll

```diff
@@ -107,7 +107,7 @@ declare <vscale x 4 x i32> @__vecz_b_sub_group_scan_exclusive_add_u5nxv4j(<vscal
 ; CHECK:   %[[SCAN:.+]] = phi <vscale x 4 x i32> [ %[[NEWVEC]], %loop ]
 
 ;------- target-dependent slide-up code:
-; CHECK:   %[[SLIDE:.+]] = call <vscale x 4 x i32> @llvm.experimental.vector.splice.nxv4i32(<vscale x 4 x i32> undef, <vscale x 4 x i32> %[[SCAN]], i32 -1)
+; CHECK:   %[[SLIDE:.+]] = call <vscale x 4 x i32> @llvm{{(.experimental)?}}.vector.splice.nxv4i32(<vscale x 4 x i32> undef, <vscale x 4 x i32> %[[SCAN]], i32 -1)
```

Comment:
In a regex, `.` means "match any character"; to match a literal `.`, it should be `\.`.